feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/autodiscovery/AccountAutoDiscoveryContent.kt

```diff
@@ -1,80 +1,187 @@
             ) {
-                contentItems(
-                    state = state,
-                    onEvent = onEvent,
-                    oAuthViewModel = oAuthViewModel,
+                Column(
+                    modifier = Modifier
+                        .weight(1f)
+                        .verticalScroll(scrollState)
+                        .imePadding(),
+                ) {
+                    AppTitleTopHeader(
+                        title = stringResource(id = R.string.account_setup_title),
+                        modifier = Modifier
+                            .padding(
+                                start = MainTheme.spacings.double,
+                                end = MainTheme.spacings.double,
+                            ),
+                    )
+                    Spacer(modifier = Modifier.weight(1f))
+                    AutoDiscoveryContent(
+                        state = state,
+                        onEvent = onEvent,
+                        oAuthViewModel = oAuthViewModel,
+                    )
+                    Spacer(modifier = Modifier.weight(1f))
+                }
+
+                WizardNavigationBar(
+                    onNextClick = { onEvent(Event.OnNextClicked) },
+                    onBackClick = { onEvent(Event.OnBackClicked) },
+                    state = WizardNavigationBarState(showNext = state.isNextButtonVisible),
+                )
+            }
+        }
+    }
+}
+
+@Composable
+internal fun AutoDiscoveryContent(
+    state: State,
+    onEvent: (Event) -> Unit,
+    oAuthViewModel: AccountOAuthContract.ViewModel,
+    modifier: Modifier = Modifier,
+) {
+    val resources = LocalContext.current.resources
+
+    ContentLoadingErrorView(
+        state = rememberContentLoadingErrorViewState(state),
+        loading = {
+            LoadingView(
+                message = stringResource(id = R.string.account_setup_auto_discovery_loading_message),
+                modifier = Modifier.fillMaxSize(),
+            )
+        },
+        error = {
+            ErrorView(
+                title = stringResource(id = R.string.account_setup_auto_discovery_loading_error),
+                message = state.error?.toAutoDiscoveryErrorString(resources),
+                onRetry = { onEvent(Event.OnRetryClicked) },
+                modifier = Modifier.fillMaxSize(),
+            )
+        },
+        content = {
+            ContentView(
+                state = state,
+                onEvent = onEvent,
+                oAuthViewModel = oAuthViewModel,
+                resources = resources,
+            )
+        },
+        modifier = Modifier
+            .fillMaxSize()
+            .then(modifier),
+    )
+}
+
+@Composable
+internal fun ContentView(
+    state: State,
+    onEvent: (Event) -> Unit,
+    oAuthViewModel: AccountOAuthContract.ViewModel,
+    resources: Resources,
+    modifier: Modifier = Modifier,
+) {
+    Column(
```

Comment:
[Optional] Instead of manually adding `Spacer` we could use `verticalArrangement = Arrangement.spacedBy(MainTheme.spacings.double)`

Reply:
The UI doesn't use the same spacing for every element.